webapp/src/ts/modules/tasks/tasks.component.ts

```diff
@@ -118,50 +128,91 @@ export class TasksComponent implements OnInit, OnDestroy {
 
   private hydrateEmissions(taskDocs) {
     return taskDocs.map(taskDoc => {
-      const emission = { ...taskDoc.emission };
+      const emission = {...taskDoc.emission};
       const dueDate = moment(emission.dueDate, 'YYYY-MM-DD');
       emission.date = new Date(dueDate.valueOf());
       emission.overdue = dueDate.isBefore(moment());
       emission.owner = taskDoc.owner;
-
+      emission.forId = taskDoc.forId ? taskDoc.forId : taskDoc.owner;
```

Comment:
Are we sure that we are not going to inadvertently trigger some additional logic by setting the `forId` to `owner`?

I am not sure exactly what the difference is between the two fields anyway, but I am a little worried that this could have unintended consequences...

Reply:
@elvisdorkenoo - To play safe, you can:
- Line 135 has `emission.owner` already defined and available. 
- Line 136 - Add `emission.forId` and don't default it to `task.owner` because it's already expose in `emission.owner`.
- In your `getLineagesFromTaskDocs` check for both properties: `const ids = [...new Set(taskDocs.map(task => task.forId || task.owner))];`

Reply:
`task => task.forId || task.owner ` is actually what I've before changing it this way when I got the confirmation that when it's set, task.forId always equals task.owner. But yeah let's play it safe.